src/main/java/frc/robot/RobotContainer.java

```diff
@@ -28,51 +30,115 @@
+    ShuffleboardLayout conveyorCommands = Shuffleboard.getTab("Commands")
+        .getLayout("Conveyor", BuiltInLayouts.kList)
+        .withSize(2, 2)
+        .withPosition(2, 0)
+        .withProperties(Map.of("Label position", "HIDDEN"));
+    conveyorCommands.add("Move 1 in. up", new MoveConveyorDistanceCommand(m_conveyor, Units.inchesToMeters(5)));
+    conveyorCommands.add("Move 1 in. down", new MoveConveyorDistanceCommand(m_conveyor, Units.inchesToMeters(-5)));
+    conveyorCommands.add("Move up", new RunConveyorCommand(m_conveyor, Direction.kUp));
+    conveyorCommands.add("Move down", new RunConveyorCommand(m_conveyor, Direction.kDown));
   }
 
+
   /**
    * Use this method to define your button->command mappings. Buttons can be created by
    * instantiating a {@link GenericHID} or one of its subclasses ({@link
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
-
-    JoystickButton rpmButton1 = new JoystickButton(m_controller, XboxController.Button.kA.value);
-    rpmButton1.whileHeld(new RunShooterCommand(m_shooter, 3000));
-
-    JoystickButton rpmButton2 = new JoystickButton(m_controller, XboxController.Button.kX.value);
-    rpmButton2.whileHeld(new RunShooterCommand(m_shooter, 4000));
-
-    JoystickButton rpmButton3 = new JoystickButton(m_controller, XboxController.Button.kB.value);
-    rpmButton3.whileHeld(new RunShooterCommand(m_shooter, 5000));
-
-    /*JoystickButton ForwardClimbClockwiseButton = new JoystickButton(m_controller, XboxController.Button.kY.value);
-    ForwardClimbClockwiseButton.whileHeld(new ForwardClimbClockwise(m_frontClimber));
-
-    JoystickButton ForwardClimbCounterClockwiseButton = new JoystickButton(m_controller, XboxController.Button.kB.value);
-    ForwardClimbCounterClockwiseButton.whileHeld(new ForwardClimbCounterClockwise(m_frontClimber));
-
-    JoystickButton BackwardClimbClockwiseButton = new JoystickButton(m_controller, XboxController.Button.kX.value);
-    BackwardClimbClockwiseButton.whileHeld(new BackwardClimbClockwise(m_backClimber));
-
-    JoystickButton BackwardClimbCounterClockwiseButton = new JoystickButton(m_controller, XboxController.Button.kA.value);
-    BackwardClimbCounterClockwiseButton.whileHeld(new BackwardClimbCounterClockwise(m_backClimber));*/
+    //Conveyor Control
+    JoystickButton conveyorUp = new JoystickButton(m_controller, XboxController.Button.kLeftBumper.value);
+    conveyorUp.whileHeld(new RunConveyorCommand(m_conveyor, Direction.kUp));
+
+    JoystickButton conveyorDown = new JoystickButton(m_controller, XboxController.Button.kRightBumper.value);
+    conveyorDown.whileHeld(new RunConveyorCommand(m_conveyor, Direction.kDown));
+    // //: Shooter control
+    // JoystickButton rpmButton1 = new JoystickButton(m_buttopad, 8);
+    // rpmButton1.whileHeld(new RunShooterCommand(m_shooter, () -> 2800));
```

Comment:
We need to re-add all of these shooter buttons. We are going to add another row of buttons on the button pad.

Reply:
74b25823dc7c4f3f6f0091066ab12013db1cc829